tests/create_LWE.py

```diff
@@ -0,0 +1,322 @@
+import numpy as np
+import subprocess
+import re
+import warnings
+from typing import NoReturn
+
+from CADETProcess.simulator import Cadet
+from CADETProcess import CADETProcessError
+from CADETProcess.processModel import (
+    ComponentSystem, FlowSheet, Process,
+    Inlet, Outlet, Cstr, GeneralRateModel,
+    TubularReactor, LumpedRateModelWithoutPores,
+    LumpedRateModelWithPores, MCT, Linear,
+    Langmuir, LangmuirLDF, BiLangmuir,
+    BiLangmuirLDF, FreundlichLDF, StericMassAction,
+    AntiLangmuir, Spreading, SelfAssociation,
+    BiStericMassAction, MultistateStericMassAction,
+    SimplifiedMultistateStericMassAction, Saska,
+    GeneralizedIonExchange
+)
+
+
+def create_lwe(unit_type: str = 'generalratemodel', **kwargs) -> Process:
+    n_comp: int = kwargs.get('n_comp', 4)
+    component_system = ComponentSystem(n_comp)
+    flow_sheet = FlowSheet(component_system)
+
+    unit_type = unit_type.lower()
+    unit = None
+
+    if unit_type == 'cstr':
```

Comment:
To make this easy for the user, right now this function ignores capitalisation, so I could either write "LumpedRateModelWithPores" or "lumpedratemodelwithpores" or "LumpedrateodelwithPores". Do we want this to be explicitly only one capitalasation?

Reply:
```
import this
The Zen of Python, by Tim Peters

Beautiful is better than ugly.
\*\*Explicit is better than implicit.\*\*
Simple is better than complex.
Complex is better than complicated.
Flat is better than nested.
Sparse is better than dense.
Readability counts.
Special cases aren't special enough to break the rules.
Although practicality beats purity.
Errors should never pass silently.
Unless explicitly silenced.
In the face of ambiguity, refuse the temptation to guess.
\*\*There should be one-- and preferably only one --obvious way to do it.\*\*
Although that way may not be obvious at first unless you're Dutch.
Now is better than never.
Although never is often better than *right* now.
If the implementation is hard to explain, it's a bad idea.
If the implementation is easy to explain, it may be a good idea.
Namespaces are one honking great idea -- let's do more of those!
```